src/app/constants/validator.ts

```diff
@@ -0,0 +1,19 @@
+export class Validator {
```

Comment:
I think it should be moved into app/validators and should be renamed to json.validator.ts
What do you think about this?